private/dump_new_fish_data.py

```diff
@@ -535,6 +536,16 @@ def is_big_fish(fish):
     return False
 
 
+def get_min_rating(fish):
+    for collectable in tracked_iter(realm.game_data.get_sheet('CollectablesShopItem'),
+                                    'Checking Collectable Shop for Scrip Fish Rating'):
+        if fish.item.name == collectable.as_string('Item'):
+            for rating_set in tracked_iter(realm.game_data.get_sheet('CollectablesShopRefine'),
+                                           'Getting rating set for fish'):
+                if collectable.as_string('CollectablesShopRefine').split('#')[1] == rating_set.key:
+                    return rating_set.as_int16('LowCollectability')
+
+
```

Comment:
This isn't how `tracked_iter` is meant to be used.
Similar solution that I suggested in the other comment should be used here.  I don't think it's necessary to iterate over the entire table since for each fish marked as collectable, we can just drill down from there.

Reply:
Mimicked other solution, honestly didn't realize it returned the entire row. Definitely my mistake there.

Reply:
Cancel that, rereading the code I think this function is very unnecessary in its entirety.